internal/cmd/base/option.go

```diff
@@ -149,3 +150,10 @@ func WithStatusCode(statusCode int) Option {
 		o.withStatusCode = statusCode
 	}
 }
+
+// WithDatabaseTemplate allows for using an existing database template for initializing the boundary database.
```

Comment:
Very small thing, but this comment is particularly long and I think most of us are wrapping comments around 80chars.

Reply:
fixed